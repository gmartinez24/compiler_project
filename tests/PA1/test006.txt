// Isuranga Perera
// Test to check expressions with whitespaces are correctly tokenized
/ / //
main

float positiveFloat = ==  12.564 .234;
float positiveFloat = - 12.48  .0;

int temp1 temp2 += 4  + 5;
mainif
main if
