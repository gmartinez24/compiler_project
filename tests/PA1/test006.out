DIV
DIV
MAIN
FLOAT
IDENT	positiveFloat
ASSIGN
EQUAL_TO
FLOAT_VAL	12.564
PERIOD
INT_VAL	234
SEMICOLON
FLOAT
IDENT	positiveFloat
ASSIGN
SUB
FLOAT_VAL	12.48
PERIOD
INT_VAL	0
SEMICOLON
INT
IDENT	temp1
IDENT	temp2
ADD_ASSIGN
INT_VAL	4
ADD
INT_VAL	5
SEMICOLON
IDENT	mainif
MAIN
IF
EOF
